lib/checks/navigation/region-evaluate.js

```diff
@@ -45,14 +45,19 @@ function getRegionlessNodes(options) {
  */
 function findRegionlessElms(virtualNode, options) {
   const node = virtualNode.actualNode;
-  // End recursion if the element is a landmark, skiplink, or hidden content
+  // End recursion if the element is...
+  // - a landmark
+  // - a skiplink
+  // - hidden content
+  // - a presentation graphic
   if (
     getRole(virtualNode) === 'button' ||
     isRegion(virtualNode, options) ||
     ['iframe', 'frame'].includes(virtualNode.props.nodeName) ||
     (dom.isSkipLink(virtualNode.actualNode) &&
       dom.getElementByReference(virtualNode.actualNode, 'href')) ||
-    !dom.isVisibleToScreenReaders(node)
+    !dom.isVisibleToScreenReaders(node) ||
```

Comment:
This brings up an interesting question. Should a presentational img using `alt=""` be considered not visible to screen readers? @WilcoFiers is the better change to update `isVisibleToScreenReaders` to return false for an img with `alt=""`? Right now `isVisibleToScreenReaders` returns true for an img with empty alt.

Reply:
You know I was wondering that, it was a path I started diving down and then uncertainty got the best of me. It's a function leveraged in a lot of places.

If we think that's the right path I'm happy to try it out and see if any other test failures come up along the way that make us second guess it.

Reply:
Fair question, but no I don't think so. Conceptually, there's a difference between a hidden element and an element that's ignored. It matters for example on `<img role="button" alt="">`. That should fail for having an inappropriate role, whereas `<img role="button" aria-hidden="true" alt="">` should be incomplete.